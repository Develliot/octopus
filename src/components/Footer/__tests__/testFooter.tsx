import React from 'react';
import { shallow, ShallowWrapper } from 'enzyme';
import { Footer } from '../';

it('renders the component properly', () => {
    const wrapper: ShallowWrapper = shallow(<Footer />);
    expect(wrapper).toMatchSnapshot();
});
