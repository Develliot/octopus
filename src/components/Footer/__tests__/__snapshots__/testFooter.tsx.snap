// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the component properly 1`] = `
<footer>
  <ColorBlock
    padding={false}
  >
    <Layout__ContentWrapper>
      <Layout__VerticalSpacer
        size="small"
      />
      <Typography__Paragraph
        fontSize="tiny"
        style={
          Object {
            "fontWeight": 200,
          }
        }
      >
        Octopus Energy Ltd is a company registered in England and Wales.
      </Typography__Paragraph>
      <Typography__Paragraph
        fontSize="tiny"
        style={
          Object {
            "fontWeight": 200,
          }
        }
      >
        Registered number: 09263424. Registered office: 33 Holborn, London, EC1N 2HT. Trading office: 20-24 Broadwick Street, London, W1F 8HT.
      </Typography__Paragraph>
      <Layout__VerticalSpacer
        size="small"
      />
    </Layout__ContentWrapper>
  </ColorBlock>
</footer>
`;
